app/models/bulkrax/exporter.rb

```diff
@@ -96,7 +96,7 @@ def importers_list
     end
 
     def current_run
-      @current_run ||= self.exporter_runs.create!(total_work_entries: self.limit || parser.total)
+      @current_run ||= self.exporter_runs.create!(total_work_entries: self.limit || parser.total, enqueued_records: self.limit || parser.total)
```

Comment:
Since we're setting both counters to the same value, this could be made slightly cleaner by doing something like this: 

```ruby
    def current_run
      total = self.limit || parser.total
      @current_run ||= self.exporter_runs.create!(total_work_entries: total, enqueued_records: total)
```

Reply:
Awesome thanks, made the changes!